pandas/core/arrays/categorical.py

```diff
@@ -1883,7 +1885,8 @@ def _validate_setitem_value(self, value):
             new_codes = self._validate_listlike(value)
             value = Categorical.from_codes(new_codes, dtype=self.dtype)
 
-        rvalue = value if is_list_like(value) else [value]
+        # wrap scalars and hashable-listlikes in list
+        rvalue = value if not is_hashable(value) else [value]
```

Comment:
can use `maybe_make_list` (but certainly can followup as may need to change that definition slightly)